tests/bson/bson-timestamp-set_state_error-004.phpt

```diff
@@ -8,19 +8,19 @@ MongoDB\BSON\Timestamp::__set_state() with broken numeric strings
 require_once __DIR__ . '/../utils/tools.php';
 
 echo throws(function() {
-    MongoDB\BSON\Timestamp::__set_state(['increment' => 'broken', 'timestamp' => '5678']);
+    MongoDB\BSON\Timestamp::__set_state(['increment' => '1.23', 'timestamp' => '5678']);
```

Comment:
This test would have been problematic for Windows since `_atoi64()` stops parsing at the first invalid character without raising an error; however, `bson_ascii_strtoll()` is more strict.